system/boot-operator-remote/values.yaml

```
@@ -1,40 +1,68 @@
+owner-info:
+  support-group: containers
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/system/boot-operator-remote
+
+fullnameOverride: boot-operator
+
+boot-operator-core:
+  rbac:
+    enable: false
+  crd:
+    enable: false
+  metrics:
+    enable: false
+  prometheus:
+    enable: false
+  networkPolicy:
+    enable: false
+  webhook:
+    enable: true
+  certmanager:
+    enable: true
+
+  controllerManager:
+    enable: true
+    pod:
+      labels:
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-private-networks: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-runtime-apiserver: allowed
+        networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443: allowed
+        networking.resources.gardener.cloud/to-kube-apiserver-tcp-443: allowed
+    manager:
+      image:
+        repository: controller
+        tag: fe855e5b44cc7132dff46087cc80ef8d60893f36
+      args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        - --controllers=httpbootconfig,ipxebootconfig,serverbootconfighttp,serverbootconfigpxe
+      env:
+        KUBERNETES_SERVICE_HOST: "apiserver-url"
+        KUBERNETES_CLUSTER_DOMAIN: "cluster.local"
+      ports:
+        - name: ipxe
+          containerPort: 8082
+      volumes:
+        - name: remote-kubeconfig
+          mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+          readOnly: true
+          source:
+            secret:
+              secretName: boot-operator-remote-kubeconfig
+              items:
+              - key: token
+                path: token
+              - key: bundle.crt
+                path: ca.crt
+      serviceAccountName: controller-manager
```

I think it needs to be `default` in remote use cases
```suggestion
      serviceAccountName: default
```

---

done